Neos.Neos/Classes/FrontendRouting/Projection/DocumentUriPathProjection.php

```diff
@@ -983,7 +982,8 @@ private function whenDimensionShineThroughWasAdded(DimensionShineThroughWasAdded
                     precedingnodeaggregateid,
                     succeedingnodeaggregateid,
                     shortcuttarget,
-                    nodetypename
+                    nodetypename,
+                    isplaceholder
```

Comment:
tiny nitpick, but I suggest to follow the trailing comma rule of [PER 2.0](https://www.php-fig.org/per/coding-style/#26-trailing-commas)
```suggestion
                    isplaceholder,
```